src/main/java/com/atlassian/db/replica/api/reason/RouteDecision.java

```diff
@@ -38,6 +38,20 @@ public Reason getReason() {
         return Optional.ofNullable(cause);
     }
 
+    /**
+     * @return True if the database call would always fail on a read replica.
+     */
+    public boolean isWrite() {
+        return reason.isWrite();
+    }
```

Comment:
```suggestion
    /**
     * @return false if the accompanying {@link SqlCall#call()} would fail when run on replica.
     */
    public boolean couldRunOnReplica() {
        return !reason.isWrite();
    }
```
Or maybe:
```suggestion
    /**
     * @return true if the accompanying {@link SqlCall#call()} would fail when run on replica.
     */
    public boolean mustRunOnMain() {
        return reason.isWrite();
    }
```

I'm trying to use symmetrical language across the `RouteDecision` API.